README.md

```diff
@@ -11,10 +11,11 @@ A utility library for working with [JSON Table Schema](http://dataprotocols.org/
 [Installation](#installation)  
```

Comment:
We need some NOTE on top of readme about breaking changes - see https://github.com/frictionlessdata/jsontableschema-py#json-table-schema